### PR TITLE
pool+order: display fee estimates in the order form

### PR DESCRIPTION
**Pull request by jamaljsr**
Based on #242 

This PR makes use of the new `QuoteOrder` RPC in Pool to display fee estimates in the order form. The values automatically update then the user modifies the order form fields. The new fee rows also have tooltips to provide more helpful information.

image

### CODE REVIEW

File: app/src/store/views/orderFormView.ts [hunk 1]
```diff
@@ -229,6 +239,39 @@ export default class OrderFormView {
...
+    );
+
+    runInAction(() => {
+      this.executionFee = totalExecutionFeeSat;
+      this.worstChainFee = worstCaseChainFeeSat;
+    });
+  }
+
+  /** quote order at most once every 2 seconds when using this func  */
+  quoteOrderThrottled = debounce(this.quoteOrder, 2000);
```
Comment (guggero) [inline app/src/store/views/orderFormView.ts:273]: This tripped me up a bit. I changed some values and expected the quote to be updated. Was almost ready to enter yet another number when I noticed the number finally updated. Maybe I'm just too impatient.

If it's easy to implement (probably not worth doing if it requires a lot of effort), we could invalidate the quote fields (for example by showing a loading animation) until the next debounce run comes along. That way the user knows something's going to happen soon.

**Review by guggero — APPROVED** [review]
tACK, LGTM :100: 

Just one small UX improvement idea that is totally non-blocking.

File: app/src/store/views/orderFormView.ts (hunk 1 above)
Comment (jamaljsr) [reply app/src/store/views/orderFormView.ts:273]: ah, good point. I like the idea of displaying a loader. I can do that quickly.

Comment (jamaljsr) [reply app/src/store/views/orderFormView.ts]: I added the loaders and decreased the debounce to 1 second. Thanks for the suggestion

**Pull request merged**